garnbarn_api/views.py

```diff
@@ -106,7 +112,8 @@ def create(self, request, *args, **kwargs):
         return Response(serializer.data, status=status.HTTP_200_OK)
 
     def perform_create(self, serializer):
-        serializer.save()
+        user_data = self.request.user.get_json_data()["uid"]
```

Comment:
Maybe this can be minimized by `self.request.user.uid`

Reply:
Done in commit bc03e83.